realm/src/main/java/io/realm/internal/Group.java

```diff
@@ -55,29 +51,25 @@ public Group() {
         checkNativePtrNotZero();
     }
 
-    protected native long createNative();
-
     public enum OpenMode {
```

Comment:
I would just get rid of the enum for these internal classes and use int constants.